hera_mc/cm_revisions.py

```diff
@@ -48,24 +43,33 @@ def get_revisions_of_type(hpn, rev_type, at_date=None, session=None):
     at_date:  astropy.Time to check for
     session:  db session, or hookup_dict if FULLY_CONNECTED
     """
+    if isinstance(hpn, six.string_types) and hpn.lower() == 'help':
+        print("""
+        Allowed revision types or revisions are (case doesn't matter and only need first 3 letters):
+            [ACT]IVE:  revisions active at_date ('now' as default) -- only one that uses 'at_date'
+            [LAS]T:  the last connected revision (could be active or not)
+            [FUL]LY_CONNECTED:  revision that is part of a fully connected signal path in current or supplied cache
```

Comment:
Here you're clearer, but you actually suggest a 3rd option `FUL`. Maybe we can make this clearer throughout?